src/twisted/web/http.py

```diff
@@ -1416,7 +1416,13 @@ def _sanitize(val):
             cookie = cookie + b"; HttpOnly"
         if sameSite:
             sameSite = _ensureBytes(sameSite).lower()
-            if sameSite not in [b"lax", b"strict"]:
+            if not secure and sameSite == b"none":
```

Comment:
Maybe add a link to the docs used to implement this

```suggestion
            # See more info about sameSite usage here
            # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value
            if not secure and sameSite == b"none":
```